internal/service/cloudbackupsnapshotexportbucket/data_source_cloud_backup_snapshot_export_bucket.go

```diff
@@ -45,7 +48,7 @@ func datasourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	conn := meta.(*config.MongoDBClient).AtlasV2
 
 	projectID := d.Get("project_id").(string)
-	bucketID := d.Get("id").(string)
+	bucketID := d.Get("export_bucket_id").(string)
```

Comment:
does this mean it was not working?

Reply:
It was working. But it was quite confusing:

The `id` it expected was: `mongodbatlas_cloud_backup_snapshot_export_bucket.test.export_bucket_id` (see changed test)
not: `mongodbatlas_cloud_backup_snapshot_export_bucket.test.id`